lib/icinga/notification.hpp

```diff
@@ -92,6 +93,8 @@ class Notification final : public ObjectImpl<Notification>
 	static String NotificationHostStateToString(HostState state);
 
 	static boost::signals2::signal<void (const Notification::Ptr&, const MessageOrigin::Ptr&)> OnNextNotificationChanged;
+	static boost::signals2::signal<void (const Notification::Ptr&, const String&, uint_fast8_t, const MessageOrigin::Ptr&)> OnLastNotifiedStatePerUserUpdated;
+	static boost::signals2::signal<void (const Notification::Ptr&, const MessageOrigin::Ptr&)> OnLastNotifiedStatesCleared;
```

Comment:
I'd also rename this to `OnLastNotifiedStatePerUserCleared`. Why? Because a simple `git grep LastNotifiedStatePerUser` will then lead you to all the places that affect this attribute.